### PR TITLE
ISO update 6.0.4

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
Updates for new 604 IBU ISO, and removed an outdated file - Services Description.

### CODE REVIEW

**Review by MichaelBaj — APPROVED** [review]

File: docs/intro_downloading_iso.md [hunk 1]
```diff
@@ -13,24 +13,26 @@ Juniper Session Smart Networking provides the following workflows for the instal
 
   For details about the Image-based install process, see [Image-based Installation.](intro_installation_image.md) 
 
-- **One Touch Provisioning (OTP):** This is the preferred method of installation. Sets up DHCP on all interfaces and boots a Web Server GUI. After installing the Conductor and configuring routers through the Conductor, the OTP bootstrap process will install and configure the router.
+- **Package-based ISO:** For users who do not use Mist Cloud, this ISO offers multiple local installation methods.
+  - **One Touch Provisioning (OTP)** is the default and preferred method of installation. OTP sets up DHCP on all interfaces and boots a Web Server GUI. After installing the Conductor and configuring routers through the Conductor, the OTP bootstrap process will install and configure the router.
```
Comment (MichaelBaj) [inline docs/intro_downloading_iso.md:17]: Suggest adding a link to the OTP workflow document.

**Review by MichaelBaj — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/intro_downloading_iso.md [hunk 2]
```diff
@@ -13,24 +13,26 @@ Juniper Session Smart Networking provides the following workflows for the instal
...
+  - **Interactive:** For bespoke deployments where customized platform configuration is necessary, an interactive mode exists. Installation is done via a serial console or VGA. An interactive session is started to configure network interfaces, passwords, node name and type, and conductor IP (if applicable) before the SSR software is started.
 
 :::note
-For bespoke deployments where customized platform configuration is necessary, an interactive mode exists.
+Beginning with release 5.4.7-7 (and [**all releases chronologically**](about_releases.mdx#all-releases---limited-and-general-availability) after August 4, 2022) the ISO name format has changed from using `OTP` to `ISO`:
```
Comment (jeffmart-jnpr) [inline docs/intro_downloading_iso.md:21]: Can we add to this that "any 5.x ISO after August 4, 2022" as the 6.x ISO do not fall in this bucket and have the old format.

File: docs/release_notes_128t_6.0.md [hunk 3]
```diff
@@ -61,6 +61,10 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 - **I95-47537/I95-47556 Synchronize writing to files to avoid a race condition:** Added a common file lock to synchronize writes.
 ------
 - **I95-47655 BGP issues with VRRP:** VRRP failover may cause routing to not function if internal device numbering is not consistent across the redundant nodes.
+------
+- **WAN-1327/MIST-69263 Device Adoption may fail:** Resolved issues where a "factory reset" system failed to be adopted. This could be seen if an invalid claim code was entered during the adoption process, or when a whitebox system (non Juniper-branded) was not successfully released.
+
+	This issue was missing from the `6.0.4-11 v7` IBU image, and has been corrected in the `SSR-6.0.4-11.el7.x86_64.ibu-v8.iso` ISO.
```
Comment (jeffmart-jnpr) [inline docs/release_notes_128t_6.0.md:67]: This needs to be "v8" not "v7"

**Review by jeffmart-jnpr — CHANGES_REQUESTED** [review]
Need 2 changes

File: docs/release_notes_128t_6.0.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md:67]: fixed

File: docs/intro_downloading_iso.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/intro_downloading_iso.md:21]: fixed

File: docs/intro_downloading_iso.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/intro_downloading_iso.md:17]: Added links to both OTP and Quickstart

**Review by jeffmart-jnpr — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

File: docs/intro_downloading_iso.md (hunk 2 above)
Comment (iwbarker) [inline docs/intro_downloading_iso.md]: "all 5.x releases"

**Pull request merged**